packages/fxa-auth-server/lib/backendService.js

```diff
@@ -217,6 +224,12 @@ module.exports = function createBackendServiceAPI(
         payload,
         this._headers
       );
+
+      // The statsD dependency is optional
+      if (statsd) {
+        statsd.timing(`${serviceName}.${methodName}`, Date.now() - startTime);
```

Comment:
IIUC, this won't happen if an error is thrown from the `sendRequest` call above?

Reply:
Yes.  I'll file a bug and create a patch.  Thanks!